### PR TITLE
Add 18 manual

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Comment by skjnldsv** [comment]
Also requires changes here https://github.com/nextcloud/documentation/blob/c502591912bbb0f141938a2857835558d7cfd710/conf.py#L40

**Comment by rullzer** [comment]
> Also requires changes here
> 
> https://github.com/nextcloud/documentation/blob/c502591912bbb0f141938a2857835558d7cfd710/conf.py#L40

Not yet. that needs to happen once it is stable ;)

**Comment by skjnldsv** [comment]
> Not yet. that needs to happen once it is stable ;)

AH, I thought this pr would be merged after stable too :)

**Comment by rullzer** [comment]
> > Not yet. that needs to happen once it is stable ;)
> 
> AH, I thought this pr would be merged after stable too :)

Nah I want it all ready and in somewhat working state before that ;)

File: index.html [hunk 1]
```diff
@@ -219,6 +219,18 @@ <h1>Nextcloud Server<a class="headerlink" href="#server" title="Permalink to thi
 								page for more information.</p>
 							</div>
 
+							<div class="section" id="nextcloud-17">
```
Comment (ChristophWurst) [inline index.html]:
```suggestion
							<div class="section" id="nextcloud-18">
```

**Pull request merged**